### DISCUSSION TITLE
Leaflet always uses the full width

### DISCUSSION THREAD

**Discussion by amks1** [zauberzeug/nicegui#2988] [Q&A]
### Description

I'm trying to fit leaflet into one third of the page width.

```python3
with ui.row().classes('w-full'):
    ui.label("Test1")
    ui.label("Test2")
    ui.leaflet()
```

Output:
![image](https://github.com/zauberzeug/nicegui/assets/68153785/356e896a-c2f8-4fae-8937-0bdd8b253504)

If I do something like this, the map doesn't show at all.

```python3
with ui.row().classes('w-full'):
    ui.label("Test1")
    ui.label("Test2")
    with ui.element('div'):
        ui.leaflet()
```

Output:
![image](https://github.com/zauberzeug/nicegui/assets/68153785/a81517ff-ba7e-48e6-9d12-2470446deccf)

**Comment by falkoschindler** [comment]
Hi @amks1, `ui.row` wraps by default. You can disable it like this:
```py
with ui.row(wrap=False).classes('w-full'):
    ui.label("Test1")
    ui.label("Test2")
    ui.leaflet()
```
